src/bin.ts

```diff
@@ -50,8 +50,8 @@ export function main() {
 }
 
 function isTsEsmLoaderNeeded(cmd: string, args: string[]) {
-  // HACK: Ugly way to check if it's ts-node instead of node
-  return [cmd, ...args].includes("ts-node");
+  // HACK: Ugly way to check
+  return [cmd, ...args].includes("ts-node") || existsSync(join(cwd(), "tsconfig.json"));
```

Comment:
I don't think this is a good idea. The presence of `tsconfig` doesn't mean `ts-node` is available or the project uses it.